_posts/2022-04-19-debt-reduction-in-the-product-roadmap.md

```diff
@@ -2,32 +2,34 @@
 layout: post
 title: "How to include refactoring into Product development roadmap? Reducing technical debt inspired by real options identification"
 author: [olga.dudzik]
-tags: [tech]
+tags: [tech, roadmaps, tech debt, product]
 ---
 
-## Nowadays, technical debt can be considered a bread and butter of most IT-powered enterprises around the world.
-
-Almost every company that survived the startup phase and managed to deliver its first products to customers will face at some point technical challenges related to past architectural decisions. Although code engineering gets better every year, we cannot argue with the obvious fact of life: the market will always force many of us to deliver tech products faster than we wish. Time To Market has always been a key success factor for many product companies and it puts a lot of pressure on Engineering to keep up to challenging deadlines.
+Nowadays, technical debt can be considered the bread and butter of most IT-powered enterprises around the world.
+Almost every company that survived the startup phase and managed to deliver its first products to customers will face at some point technical challenges related to past architectural decisions. Although code engineering gets better every year, we cannot argue with the obvious fact of life: the market will always force many of us to deliver tech products faster than we wish. Time To Market has always been a key success factor for many product companies and it puts a lot of pressure on Engineering to keep up with challenging deadlines.
 
 Statistics explicitly show the scale of the problem. According to the survey conducted in 2020 by McKinsey[^1], tech debt can reach up to as much as 40% of the whole technology value. On average 10-20% of IT budget is ultimately consumed by tech debt management and most CIOs interviewed consider the problem significantly increasing over past years, especially in enterprise-size companies[^2].
 
 [^1]: https://www.mckinsey.com/business-functions/mckinsey-digital/our-insights/tech-debt-reclaiming-tech-equity
```

Comment:
I like the idea of these links, but when the page is rendered they are not clickable. I had to copy the _URL_ to visit the mentioned page :pensive:
This may be a limitation of _Jekyll_ that we use for generating our blog. I think we have to change these links to be clickable until we configure it properly. Or maybe @mkosmul knows a workaround? :wink:

These superscript links lead to the footnotes.
<img width="1130" alt="image" src="https://user-images.githubusercontent.com/15714183/171042814-d06106db-e1b0-4b02-ae62-86fd568c7321.png">

These footnotes lead to their superscript links, not to the actual external websites. The _URLs_ are not clickable.
<img width="876" alt="image" src="https://user-images.githubusercontent.com/15714183/171042899-92bf2f3d-f133-490b-8390-a096f79f261b.png">